### ISSUE TITLE
[RFE] Kubernetes jargon, plain language, and glossary descriptions

### ISSUE THREAD

**Issue by illume** [kubernetes-sigs/headlamp#874]
## Current situation

Kubernetes jargon is confusing and extensive. Making it hard to understand for users new and long term.

Like many usability issues, it's also an accessibility issue. See WCAG 2.1, which has requirements for plain language and has guidelines for Jargon.

For languages other than English we are currently using the English Kubernetes terms. This is for two reasons. Firstly there aren't always appropriate words, so the English word can be better. Secondly, even when there are words available they can be confusing when much of the tooling using the English terms only.

## Impact

Headlamp (and Kubernetes) will become more usable and accessible.

## Ideal future situation

There will be a way for K8s Jargon words to be understood by people without extensive Kubernetes jargon knowledge.

## Implementation options

WCAG 2.1 and has some techniques about what to do when things like jargon are used. https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=313#unusual-words

Kubernetes maintains a glossary here: https://kubernetes.io/docs/reference/glossary/?all=true
Some parts are translated, but not all that much. The French translation is https://kubernetes.io/fr/docs/reference/glossary/?all=true is one of the more complete glossary translations, but still misses much.

The translations inside Headlamp also have a list of non-translated k8s English terms. The existing Documentation tab should be considered to see if it can provide other language information.

## Additional information

...

**Comment by ashu8912** [comment]
@illume I would like to work on this one with you, this will be a learning opportunity for me about a11y and i18n

**Comment by blixtra** [comment]
@illume Can you provide a few examples?

I like the spirit of the issue but I'm having a hard time thinking of example.
